src/doom/p_spec.c

```diff
@@ -509,24 +509,37 @@ P_CrossSpecialLine
 
     line = &lines[linenum];
     
-    //	Triggers that other things can activate
+    if (gameversion <= exe_doom_1_2)
+    {
+        if (line->special > 98 && line->special != 104)
+        {
+            return;
+        }
+    }
+    else
+    {
+        //	Triggers that other things can activate
+        if (!thing->player)
+        {
+            // Things that should NOT trigger specials...
+            switch(thing->type)
+            {
+                case MT_ROCKET:
+                case MT_PLASMA:
+                case MT_BFG:
+                case MT_TROOPSHOT:
+                case MT_HEADSHOT:
+                case MT_BRUISERSHOT:
+                    return;
+                    break;
```

Comment:
`break;` unnecessary after `return;`.

Reply:
Fixed in https://github.com/chocolate-doom/chocolate-doom/pull/1205/commits/77f49e5ea5aae45fe904306a9364018c0fc23b71 .